Utilities/DataLoading/EdFi.LoadTools/Engine/ResourcePipeline/SubmitResource.cs

```diff
@@ -37,9 +37,7 @@ public async Task<ApiLoaderWorkItem> ProcessAsync(ApiLoaderWorkItem resourceWork
                     resourceWorkItem.Json, resourceWorkItem.ElementName,
                     resourceWorkItem.ResourceSchemaName, refreshToken).ConfigureAwait(false))
                 {
-                    if (response.StatusCode.Equals(HttpStatusCode.Unauthorized)
-                        && response.ReasonPhrase.Equals("Invalid token")
```

Comment:
I'm fine with not locking down this client code for checking the ReasonPhrase, but this indicates we made a breaking change.  The OAuth spec states the ReasonPhrase here should be "Unauthorized", which is what we're doing now, but previously we were returning the OAuth error code as the ReasonPhrase.  @vimayya Is this difference important enough to correct for backwards compatibility?  If so we'll need a new ticket.

v5.0
Status: 401 Invalid token

v5.1
Status: 401 Unauthorized

Reply:
I am fine with change of reason phrase to match the spec. There are some other error description changes as well in 5.1.0 coming from .net core and overall guidance is to go by the error code rather than the description.